### PR TITLE
Add CHANGELOG and increment version to 0.19.2

### PR DESCRIPTION
**Pull request by twittner**
Prepare bugfix release of `libp2p-ping`.

### CODE REVIEW

File: protocols/ping/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# 0.19.2
```
Comment (tomaka) [inline protocols/ping/CHANGELOG.md:1]: A date wouldn't be a bad idea I think.

```suggestion
# 0.19.2 (2002-06-18)
```

**Comment by tomaka** [comment]
If I understand correctly, if tomorrow we wanted to release a new minor version of, say, `libp2p-kad`, we would release it under `0.19.1`, is that right?

Consequently, we wouldn't tag releases anymore, I suppose?

File: protocols/ping/CHANGELOG.md (hunk 1 above)
Comment (mxinden) [reply protocols/ping/CHANGELOG.md:1]: "2020-06-18"

**Comment by romanb** [comment]
> If I understand correctly, if tomorrow we wanted to release a new minor version of, say, libp2p-kad, we would release it under 0.19.1, is that right?

I think that is the idea - any subproject / crate can be released independently (except for necessary releases of dependencies, if any).

> Consequently, we wouldn't tag releases anymore, I suppose?

What is the problem with tags of the form `libp2p-ping-v0.19.2`?

**Pull request merged**